plugins/modules/aci_l4l7_service_graph_template.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# Copyright: (c) 2025, Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_service_graph_template
+short_description: Manage L4-L7 Service Graph Templates (vns:AbsGraph)
+description:
+- Manage Layer 4 to Layer 7 (L4-L7) Service Graph Templates on Cisco ACI fabrics.
+options:
```

Comment:
did you check all the options possible from UI? in model I see more options filterBetweenNodes, svcRuleType, type

Reply:
should we check every class and latest version of the model?

Reply:
Done